src/command/output.rs

```diff
@@ -70,25 +70,36 @@ impl RoverStdout {
                         "N/A".to_string()
                     };
 
-                    table.add_row(row![subgraph.name, url, formatted_updated_at]);
+                    subgraph_info.push((subgraph.name.to_string(), url, formatted_updated_at));
                 }
 
-                println!("{}", table);
-                println!(
+                if atty::is(Stream::Stdout) {
+                    let mut table = table::get_table();
+
+                    // bc => sets top row to be bold and center
+                    table.add_row(row![bc => "Name", "Routing Url", "Last Updated"]);
+
+                    for (name, url, updated_at) in &subgraph_info {
+                        table.add_row(row![name, url, updated_at]);
+                    }
+
+                    println!("{}", table);
+                } else {
+                    println!("Name, Routing Url, Last Updated");
+                    for (name, url, updated_at) in &subgraph_info {
+                        println!("{}, {}, {}", name, url, updated_at)
```

Comment:
I think a lot of csv parsers would keep this additional space in the parsed output, right?
```suggestion
                    println!("Name,Routing Url,Last Updated");
                    for (name, url, updated_at) in &subgraph_info {
                        println!("{},{},{}", name, url, updated_at)
```

Reply:
yeah i think you're right, shouldn't have a space here.